src/pages/verify/integrations/firebase.mdx

```diff
@@ -0,0 +1,57 @@
+---
+product: verify
+category: Integrations
+sort: 3
+title: Firebase Authentication with Identity Platform
+subtitle: This tutorial demonstrates how to integrate Criipto Verify with Firebase Authentication with Identity Platform.
+---
+
+import Layout from '../../../layouts/mdx';
+export default Layout;
+
+## Firebase Authentication with OpenID Connect
+
+Firebase provides two options for authenticating users: the simpler Firebase Auth and the more comprehensive Firebase Authentication with Identity Platform.
```

Comment:
doesn't the basic firebase authentication also support OIDC?

Reply:
No it doesn't, you have to upgrade to Firebase Authentication with Identity Platform: https://firebase.google.com/docs/auth/web/openid-connect

Reply:
> No it doesn't, you have to upgrade to Firebase Authentication with Identity Platform: https://firebase.google.com/docs/auth/web/openid-connect

Ah, I see.